### PR TITLE
Escape pod tgui

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

yay
## Why It's Good For The Game

looks good

## Changelog
:cl:
add: New escape pod tgui
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/EscapePod.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <Section title="Escape Pod">
+          Welcome to Nanotransens least luxurious survival pod!
+          Have a pleasant stay!
+          <Box
+            width="100%"
+            textAlign="center">
+            <Button.Confirm
+              m="50"
+              content="Launch evacuation pod"
+              color="red"
```
Comment (psykzz) [inline tgui/packages/tgui/interfaces/EscapePod.js:20]:
```suggestion
              disabled={data.can_launch}
              color="red"
```

Comment (psykzz) [reply tgui/packages/tgui/interfaces/EscapePod.js:20]: If not just remove the `ui_data` section

**Comment by spookydonut** [comment]
Do we really want to make it slower to launch an escape pod?

I mean tgui is cool but if i'm rushing into a pod and theres a hostile on me i'd rather it just launch soon as i click the control panel

**Comment by TiviPlus** [comment]
I havent made it slower at all

It just used a html menu before so this should make it faster actually

**Pull request merged**